src/components/DoctorCard/Shared/DoctorLinks.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+
+import { Typography } from '@mui/material';
+import * as Icons from 'components/Shared/Icons';
+
+import * as Styled from '../styles';
+import * as Links from './Links';
+
+const DoctorLinks = function DoctorLinks({ links, isWebsite }) {
+  const phoneLinks = links.map((phone, index, arr) => {
```

Comment:
names phoneLinks and phone are misleading here. links can contain phone numbers or urls.